### PR TITLE
feat: adding blacklist entries

### PR DESCRIPTION
**Pull request by danyoung93**

### CODE REVIEW

File: app/controllers/services/v1/blacklist_entries_controller.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+module Services
+  module V1
+    class BlacklistEntriesController < ServicesController
+
+      # GET /services/v1/blacklist_entries
+      def index
+        render json:
+          BlacklistEntryPresenter.collection_for_microservices(
+            BlacklistEntry.includes(:account).active
```
Comment (jackregnart) [inline app/controllers/services/v1/blacklist_entries_controller.rb:11]: There's no `#active` scope on the model?

File: app/models/blacklist_entry.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  #
+  validates_with BlacklistEntryValidator
+
+  #
+  # Relationships
+  #
+  belongs_to :account
+
+  # Scopes
+  scope :archived, -> { where(archived: true) }
```
Comment (jackregnart) [inline app/models/blacklist_entry.rb:15]: Do you want to make one called `:active` that is the inverse of this query?

File: app/validators/blacklist_entry_validator.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class BlacklistEntryValidator < ActiveModel::Validator
+  def validate(record)
+    return if (record.ip_literal.present? && record.ip_literal != "") ||
```
Comment (jackregnart) [inline app/validators/blacklist_entry_validator.rb:5]: The method `#present?` checks for a `nil` as well as a non-empty string. You don't need to do this. Just call `#present?`.

Comment (danyoung93) [reply app/validators/blacklist_entry_validator.rb:5]: Oooh that's cool!

**Pull request merged**